### DISCUSSION TITLE
{Solved} Shelly 1: Error OTA-Update to 12.4.0 - Failed HTTP error: connection failed

### DISCUSSION THREAD

**Discussion by GerdRuetten** [arendst/Tasmota#17980] [Q&A] [answered]
hello community,
I have 20 Shellys that have been running on Tasmota 12.3.1 (DE) so far.
I was able to update 19 of them to 12.4.0 (DE) without errors using OTA update.
With a Shelly, model Shelly 1, the OTA update always aborts with the following error message:

`07:47:21.029 MQT: stat/tasmota_6A3624/UPGRADE = {"Upgrade":"Failed HTTP error: connection failed"}`

The OTA URL is set correctly in my opinion:
`07:54:50.039 MQT: stat/tasmota_6A3624/RESULT = {"OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota-DE.bin.gz"}`

Same error with OTA URL:
`http://ota.tasmota.com/tasmota/release/tasmota.bin.gz`

The WiFi connection is also good:
`AP1 SSID (RSSI) FRITZ!Box 7590 RU (88%, -56 dBm) 11n`

I can't find the error. Where else can I search?

TIA

Best regards

Gerd

**Comment by barbudor** [comment, marked as answer]
On this device, what would be the result of
`WebQuery http://ota.tasmota.com/tasmota get`

If it fails, check your ip settings on this device. It could be a dns or gateway missing

Edit: better with WebQuery

**Reply by GerdRuetten** [reply]
Hi @barbudor 

thank you for your support!

08:47:06.292 CMD: WebQuery http://ota.tasmota.com/tasmota get
08:47:16.508 MQT: stat/tasmota_6A3624/RESULT = {"WebQuery":"Connect failed"}

How can i see the IP settings?

Best regards

**Comment by GerdRuetten** [comment]
OK solved.

With "Status 5" I saw that I had a wrong DNS1 entry.

I changed it to "ipaddress4 1.1.1.1", restarted the device and the update went straight through without any errors.

Thanks again @barbudor 

Perfect supprot

Best regards

Gerd
